core/src/main/java/oracle/weblogic/deploy/util/WLSDeployArchive.java

```diff
@@ -1618,6 +1735,13 @@ private static void validateExistingFile(File file, String argName, String fileN
         LOGGER.exiting(CLASS, METHOD);
     }
 
+    private static void addToRemoteMap(String localName, String newName, String type) {
+        Map<String, String> remoteMap = new HashMap<>();
+        remoteMap.put(REMOTE_TYPE, type);
+        remoteMap.put(REMOTE_ARCHIVE_DIR, newName);
+        remoteList.put(localName, remoteMap);
+    }
```

Comment:
Again, this feels wrong.  Why should the `WLSDeployArchiveFile` keep track of this map of user actions?  This is a discovery-only map for reporting discovery results to the user.  It does not really belong here.